apps/demos/Demos/Splitter/Overview/React/PaneContent.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useMemo } from 'react';
+
+const dimensionOptions = new Set(['size', 'minSize', 'maxSize'])
+
+export default function PaneContent({data}) {
+  
+  const getPaneState = (data)=>{
```

Comment:
Let's move this method out of function

Reply:
[95b4455](https://github.com/DevExpress/DevExtreme/pull/26972/commits/95b445585348fd643d71fb642b9abc659519e77c)